### ISSUE TITLE
Loading the HTML after cloning repo

### ISSUE THREAD

**Issue by kaumaron** [shhudspeth/Lahman-BaltimoreParks#2]
I tried loading the html file to view the maps and I ended up getting the Expired Key Map Error

https://developers.google.com/maps/documentation/javascript/error-messages#expired-key-map-error

(Just an FYI.)

You might have generated a new API token at some point after which might be part of it.

**Comment by shhudspeth** [comment]
Yes! I forgot to delete before I commited so I changed my API for security reasons!
